rust/src/error.rs

```diff
@@ -32,6 +32,8 @@ pub enum BackendError {
     EndOfSequence,
     #[error("WASI-NN Backend Error: Context Full")]
     ContextFull,
+    #[error("WASI-NN Backend Error: Prompt Too Long")]
+    PromptTooLong,
```

Comment:
Do we need to specify the corresponding `i32` value here?

Reply:
Implement `From<i32>` is enough for the int value mapping. Just tested on my local plugin.